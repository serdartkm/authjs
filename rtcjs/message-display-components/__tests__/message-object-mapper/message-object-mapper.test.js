import React from 'react'
import renderer from 'react-test-renderer'
import MessageObjectMapper from '../../lib/message-object-mapper/message-object-mapper'

describe("SNAPSHOT TEST MessageObjectMapper", () => {

    it("MessageObjectMapper renders correctly", () => {

        const tree = renderer
            .create(<MessageObjectMapper datetime={1569393960355} side="right" message="hello" letter="F" order="F" local={true} dateSpace={true} />).toJSON()

        expect(tree).toMatchSnapshot();
    })

})
