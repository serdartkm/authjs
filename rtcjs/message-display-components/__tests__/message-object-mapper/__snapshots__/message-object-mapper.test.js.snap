// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SNAPSHOT TEST MessageObjectMapper MessageObjectMapper renders correctly 1`] = `
<div>
  <div
    style={
      Object {
        "display": "flex",
      }
    }
  >
    <div
      style={
        Object {
          "flex": "1",
        }
      }
    >
      <hr />
    </div>
    <div>
      9/25/2019
    </div>
    <div
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <hr />
    </div>
  </div>
  <div
    style={
      Object {
        "display": "flex",
        "justifyContent": "flex-end",
      }
    }
  >
    <div
      style={
        Object {
          "alignItems": "center",
          "display": "flex",
        }
      }
    >
      <div
        style={
          Object {
            "backgroundColor": "#FFECB3",
            "borderColor": "#9E9E9E",
            "borderRadius": 15,
            "borderStyle": "solid",
            "borderWidth": 2,
            "margin": 2,
            "maxWidth": "100%",
            "minWidth": "30%",
            "padding": 5,
            "wordBreak": "break-all",
            "wordWrap": "break-word",
          }
        }
      >
        <div>
          hello
        </div>
        <div
          style={
            Object {
              "fontSize": 10,
              "paddingTop": 2,
              "textAlign": "end",
            }
          }
        >
          <i
            style={
              Object {
                "backgroundColor": "#efebe9",
              }
            }
          >
            11:46:00 AM
          </i>
        </div>
      </div>
    </div>
  </div>
</div>
`;
